src/Tests/Tests/Modules/SnapshotAndRestore/Repositories/CleanupRepository/CleanupRepositoryApiTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Elasticsearch.Net;
+using Nest;
+using Tests.Core.ManagedElasticsearch.Clusters;
+using Tests.Framework.EndpointTests;
+using Tests.Framework.EndpointTests.TestState;
+
+namespace Tests.Modules.SnapshotAndRestore.Repositories.CleanupRepository
+{
+	public class CleanupRepositoryApiTests
+		: ApiTestBase<ReadOnlyCluster, CleanupRepositoryResponse, ICleanupRepositoryRequest, CleanupRepositoryDescriptor, CleanupRepositoryRequest>
```

Comment:
Only unit tests, it'd be better but significantly more work to change the SnapshotCrud tests over to CoordinatedTestsBase so that we can insert cleanup tests controlled.

Mind adding a TODO on this so we can follow up at some point?

Reply:
updated TODO, waiting for CI before merging this in.